gui/default/syncthing/core/syncthingController.js

```diff
@@ -395,11 +395,80 @@ angular.module('syncthing.core')
                 && guiCfg.authMode !== 'ldap'
                 && !guiCfg.insecureAdminAccess;
 
+            // If no authentication configured, give info to user,
+            // in case of conflict we have time difference between openNoAuth and NoAuth.
+            $scope.NoAuth = initAuthorization(guiCfg);
+
             if (!hasConfig) {
                 $scope.$emit('ConfigLoaded');
             }
         }
 
+        function initAuthorization(guiCfg) {
+            var userPwd = !guiCfg.user || !guiCfg.password;
+
+            if (userPwd === true){
+                if (!getStorage("initNotification")){
+                    setStorage("initNotification", "1");
+                }
+
+                if (getStorage("initNotification") === "1"){
+                    setStorage("initNotification", "0");
+                    return userPwd;
+                }
+            } else {
+                removeStorage("initNotification")
+                return userPwd;
+            }
+        }
+
+        function removeStorage(key) {
+            // Removes a key from localStorage with binded expiration time.
+            try {
+                localStorage.removeItem(key);
+                localStorage.removeItem(key + '_expiresTime');
+            } catch(e) {
+                console.log('removeStorage: Error removing key ['+ key + '] from localStorage: ' + JSON.stringify(e) );
+                return false;
+            }
+            return true;
+        }
+
+        function getStorage(key) {
```

Comment:
I think these timed storage things are overkill. We have other places in here that gets and sets local storage variables, essentially just

```
try {
  window.localStorage[key] = val;
} catch (exception) { }
```

and

```
try {
  val = window.localStorage[key];
} catch (exception) { }
```

Reply:
Thanx, I made some changes base on your comments.